### PR TITLE
Replace TOML parser to support 4.0 syntax.

### LINKED ISSUE #102: some styles of header definition (to add to a redirect) do not work as expected in netlify.toml (only when parsed by netlifyctl)
**CONTEXT:**
Unable to use the following in a netlify.toml file:
```
[[redirects]]
  from = "/map"
  to = "https://maps.google.com"
  status = 200
  force = true
  headers = {X-From = "netlify_live", x-api-key= "LIVE API KEY"}
```

This causes the following error when running `netlifyctl deploy`

>"Near line 13 (last key parsed 'redirects.headers'): Expected value but found '{' instead."

**WORKAROUND:**

The following format, however, works:

```
[[redirects]]
  from = "/calculator/test"
  to = "https://futuregerald-requestbin.herokuapp.com/pzyoldpz"
  status = 200
  force = true
  [redirects.headers]
    X-From = "netlify_test"
    x-api-key= "TEST API KEY"
```

### PR DESCRIPTION
**Pull request by calavera**
Fixes #102

Signed-off-by: David Calavera <david.calavera@gmail.com>

### CODE REVIEW

**Comment by brycekahle** [comment]
We should probably update netlify-commons too right?

**Comment by calavera** [comment]
@brycekahle yeah, but that's independent from this change.

File: configuration/configuration.go [hunk 1]
```diff
@@ -51,7 +51,16 @@ func Load(configFile string) (*Configuration, error) {
...
+	f, err := os.Open(configFile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return &c, nil
+		}
+		return nil, err
+	}
+	defer f.Close()
+
+	fi, err := f.Stat()
```
Comment (brycekahle) [inline configuration/configuration.go]: This `.Stat` is probably not needed anymore since you cannot `os.Open` on a directory.

**Comment by brycekahle** [comment]
It looks like v0.3.0 of `BurntSushi/toml` supports TOML v0.4.0

**Pull request merged**